packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/pre_test.sh

```diff
@@ -14,3 +14,33 @@ test_not_val "$(agd q vstorage data published.wallet.$GOV3ADDR -o json | jq -r .
 
 # test that we have no vaults
 test_val "$(agd q vstorage data published.vaultFactory.manager0.vaults.vault0 -o json | jq -r .value)" "" "ensure no vaults exist"
+
+# provision pool has right balance
+test_val "$(agd query bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | first | .amount ')" "19250000"
```

Comment:
Note that this will likely be different if we add more actions that can affect this. At the moment it's hardcoded, but it will get cumbersome once we have many operations that affect it.

Reply:
yeah. there's a number of cumbersome/hardcoded values that could benefit in the future from use of variables to either set known constants or to hold the value of stuff retrieved pre-upgrade.